src/components/SettingsDialog/Appearance/Appearance.tsx

```diff
@@ -1,18 +1,22 @@
 import classNames from "classnames";
 import {useTranslation} from "react-i18next";
+import {getColorClassName} from "constants/colors";
+import {MenuItem} from "constants/settings";
+import {useOutletContext} from "react-router";
 import {LanguageSettingsDropdown} from "../Components/LanguageSettingsDropdown";
 import "../SettingsDialog.scss";
-import "./Appearance.scss";
 import {ThemeSettings} from "../Components/ThemeSettings";
 import {NotificationSettings} from "../Components/NotificationSettings";
 import {BoardReactionsSettings} from "../Components/BoardReactionsSettings";
 import {SkinToneSelector} from "../Components/SkinToneSelector";
+import "./Appearance.scss";
 
 export const Appearance = () => {
   const {t} = useTranslation();
+  const activeMenuItem: MenuItem = useOutletContext();
 
   return (
-    <div className={classNames("settings-dialog__container", "accent-color__lean-lilac")}>
+    <div className={classNames("settings-dialog__container", getColorClassName(activeMenuItem?.color))}>
```

Comment:
well, since the color of the menu items is determined by the menu item config, it makes sense for the actual settings dialogs to use the same color instead of using hard-coded values.